VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -294,8 +285,7 @@ protected void onOnline() {
     @Override
     protected void onOffline() {
         super.onOffline();
-        showErrorMessage(getString(R.string.no_connectivity),
-                getString(R.string.network_not_connected), false);
+        showErrorMessage(getString(R.string.no_connectivity), getString(R.string.network_not_connected));
```

Comment:
Likewise, I don't think a dialog should popup if the user goes offline, it should only appear as a result of the user submitting the form. We can probably just delete this override and the `onOnline` override.